applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -257,13 +268,14 @@ int bt_le_audio_tx_send(struct le_audio_tx_info *tx, uint8_t num_tx,
 		}
 		common_interval = tx[i].cap_stream->bap_stream.qos->interval;
 
+		struct net_buf *audio_buf = audio_frame->data;
```

Comment:
When is `audio_buf` unref'ed?

Reply:
It is unref'ed in the same function/thread the allocated it. I've tried to keep that consistent so that it doesn't matter what modules this passes by on it's way.